pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -134,6 +134,12 @@ type VitessReplicationSpec struct {
 	//
 	// Default: Semi-sync is not enforced.
 	EnforceSemiSync bool `json:"enforceSemiSync,omitempty"`
+
+	// InitializeMaster specifies whether to choose an initial master for a
+	// new or restored shard that has no master yet.
```

Comment:
Because the VitessReplicationSpec is used in shard, keyspace and cluster specs, I would advise maybe changing the comment a bit so it makes sense no matter where it gets used.
```suggestion
	// InitializeMaster specifies whether to elect an initial master for
	// new or restored shards that have no master yet.
```